content/pt/docs/referencia-da-api/pix/_index.md

```diff
@@ -1,29 +1,28 @@
 ---
-title: "PIX (Alpha)"
-linkTitle: "PIX (Alpha)"
-date: 2020-09-17T18:00:00-03:00
```

Comment:
hey @agabiandrade , I've noticed that u removed from some contents the creation date. This is not a big issue since it's only metadata, but wouldn't be nice if we could keep track of this?